### ISSUE TITLE
Righthand-nav visually odd now

### ISSUE THREAD

**Issue by larseggert** [ietf-tools/datatracker#3798]
### What happened?

There were some recent changes in feat/bs5 that allowed pulling in extra content (mainly for the agenda page).

The resulting DOM structure for the righthand-nav now seem somewhat odd. There are `card` elements in there whose purpose I don't understand and that add add visual elements (borders, spacing) that wasn't there before.

More importantly though, what's being added to the nav isn't actually shown in a useful way, as it's being horizontally truncated. See below for a comparison between 3234f1a12 (top) and 3619d3bfbf4b48dbce24a4bc3f09f02fe62c1cb0 (bottom, current):

<img width="226" alt="Screen Shot 2022-04-08 at 11 49 56" src="https://user-images.githubusercontent.com/200328/162400865-504105f5-9a48-4e19-b6a3-507aacfab8f2.png">
<img width="240" alt="Screen Shot 2022-04-08 at 11 47 57" src="https://user-images.githubusercontent.com/200328/162400889-74531bed-d018-4972-b4a6-fac6eeefee14.png">

### What browser(s) are you seeing the problem on?

Not Applicable

### Code of Conduct

- [X] I agree to follow the IETF's Code of Conduct

**Comment by rjsparks** [comment]
To be clear, this happens as the window narrows.
If the window narrows a little more, the righthand-nav is hidden.
The trigger for that hiding needs to be taken into account.

The PR that brought the cards in explained them, I think, (ability to add things like showing the selected timezone) but Jennifer can provide more detail here.

**Comment by larseggert** [comment]
As the column narrows, the contents should wrap - this is what happened before (you can see that the headings span two lines in the first image).

I'll look at where the cards came from. But the vertical centering also doesn't make sense to me.

**Comment by jennifer-richards** [comment]
The vertical centering is accidental - possibly from merge conflicts or a commit error. It can be fixed by adding an `align-items-start` to the `#righthand-nav`.

The other glitches are side effects of the timezone picker behaving badly. If I drop that (and add the `align-items-start`):
![Screenshot from 2022-04-08 11-18-14](https://user-images.githubusercontent.com/19472766/162454636-68cd72e6-32f6-4258-a9ee-feeacd37e4d4.png)

**Comment by jennifer-richards** [comment]
Re: the card, it's not essential to the changes - I personally like the padding/borders that it creates, but I think it's an aesthetic question. It could be dropped without much impact.

**Comment by jennifer-richards** [comment]
Wrapping the timezone picker turns out to be straightforward, giving this behavior:
![Screenshot from 2022-04-08 11-44-30](https://user-images.githubusercontent.com/19472766/162462699-0fbc7f20-ed53-48e7-95fa-e02115910d96.png)

**Comment by larseggert** [comment]
This is much more useful, thanks! Could I as - pretty please - to remove the card (i.e., the border and extra spacing)?

**Comment by jennifer-richards** [comment]
Working on it, will post in a minute

**Comment by jennifer-richards** [comment]
No cards.

![Screenshot from 2022-04-08 11-53-22](https://user-images.githubusercontent.com/19472766/162465135-8ff888e6-f496-46d2-84f8-187cab184dbc.png)

**Comment by jennifer-richards** [comment]
Also, I found the cause of the vertical alignment - we'd both taken the inline style off of the `#righthand-nav` and moved to CSS. I'd dropped the `70vh` height in my branch, but it came back when I merged your branch in. Without it, the panel shrinks instead of padding its content.

I've added a couple more tweaks: shrinking the size of the compact timezone picker and adding a max-height of 80vh to the entire righthand panel. At very small browser heights, this largely avoids the right hand panel from falling off the bottom of the viewport.

Reasonable browser height:
![Screenshot from 2022-04-08 13-00-17](https://user-images.githubusercontent.com/19472766/162479935-b9b8a4fb-c8f5-4e27-b347-73e16151f5a1.png)

Browser too short but everything is still nominally usable
![Screenshot from 2022-04-08 13-04-32](https://user-images.githubusercontent.com/19472766/162480161-fc728098-6189-411f-82b4-9736df96214b.png)

We could perhaps hide the time zone selector in this mode, but this isn't really an issue until the nav panel is already overflowing into a scroll bar. Regardless, usability is minimal, so I'm inclined to keep it simple.
